cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,42 @@
-describe('', () => {
-  before(() => {
+/// <reference types='cypress' />
+import { generateUser, generateeArticle } from '../support/generate';
```

Comment:
There is a typo in the import statement. It should be `generateArticle` instead of `generateeArticle`.